homework-09/script.js

```diff
@@ -0,0 +1,82 @@
+// Вивести на сторінку в один рядок через кому числа від 10 до 20
+let range = "";
+for (let i = 10; i <= 20; i++) {
+    if (i === 20) {
+        range = range + i;
+        break;
+    }
+    range = range + i + ", ";
```

Comment:
```suggestion
    range = `${range}${i}, `;
```
https://github.com/airbnb/javascript#es6-template-literals